### PR TITLE
Add more methods

### PR DESCRIPTION
**Pull request by marcus-at-localhost**
- `flattenImages`
- `getImageColorspace`
- `getImageCompressionQuality`
- `getImageDepth`
- `getImageMimeType`
- `getImageProperties` (not complete)
- `setImageCompressionQuality`

Put `$filename` in quotes in `buildConvertCommand`

### CODE REVIEW

File: src/Imagick.php [hunk 1]
```diff
@@ -1673,9 +1705,31 @@ public function getImageProfiles($pattern = "*", $only_names = true)
...
+        $current = key($files);
+
+        // TODO: implement $only_names
+        $files[$current][] = new Argument('format', '%[' . $pattern . ']');
+        $files[$current][] = 'info:';
+
+        $convert_command = $this->buildConvertCommand($files, '-');
+
+        $properties = shell_exec($convert_command);
+        #var_dump($properties);
```
Comment (calcinai) [inline src/Imagick.php]: Can you please remove this!

File: src/Imagick.php [hunk 2]
```diff
@@ -2186,10 +2240,10 @@ public function newImage($cols, $rows, $background, $format)
     {
         $argument = new Argument('size', $cols.'x'.$rows);
         $this->addConvertArgument($argument);
-        
-        $background = new ImagePickel($background);
+
+        $background = new ImagePixel($background);
```
Comment (calcinai) [inline src/Imagick.php:2242]: Ha

File: src/Imagick.php [hunk 3]
```diff
@@ -1450,7 +1454,16 @@ public function getImageColors()
...
-        throw new Exception(sprintf('%s::%s not implemented', __CLASS__, __FUNCTION__));
+        $files = $this->files;
+        $current = key($files);
+        $files[$current][] = new Argument('format', '%[colorspace]');
+        $files[$current][] = 'info:';
+
+        $convert_command = $this->buildConvertCommand($files, '-');
+        $colorspace = shell_exec($convert_command);
+
+        $name = 'Imagick::COLORSPACE_'.strtoupper($colorspace);
```
Comment (calcinai) [inline src/Imagick.php:1465]: Is this intended to be used as a constant elsewhere, or just to be descriptive?

Comment (marcus-at-localhost) [reply src/Imagick.php:1465]: Not sure if I understand, [`getImageColorspace`](https://www.php.net/manual/en/imagick.getimagecolorspace.php) returns an Int and you already provided the constants (I just looked how you did it in another method :D)

**Comment by calcinai** [comment]
Thanks @marcus-at-localhost

**Pull request merged**